compiler/src/utils/string_utils.re

```diff
@@ -13,6 +13,13 @@ let starts_with = (string, prefix) => {
   };
 };
 
+let trim_each_line = str => {
+  str
+  |> Str.split(Str.regexp("[\n\r]"))
```

Comment:
Could this give us issues on Windows, since this would end up adding an extra newline for each line, since there's a split for both `\r` and `\n`? If so, I think this would fix:

```suggestion
  |> Str.split(Str.regexp("\\(\r\n\\|\n\\)"))
```

Reply:
Yep! I wrote this extremely blurry-eyed and even then knew something was wrong, but didn't know how to fix it. Thanks! Should I add a test?

Reply:
Probably a good idea!